src/components/AccountSwitcher.tsx

```diff
@@ -0,0 +1,202 @@
+                interactive={canSwitchAccounts}
+                wrapperStyle={[styles.flexGrow1, styles.flex1, styles.mnw0, styles.justifyContentCenter]}
+            >
+                <View style={[styles.flexRow, styles.gap3]}>
+                    <Avatar
+                        type={CONST.ICON_TYPE_AVATAR}
+                        size={CONST.AVATAR_SIZE.MEDIUM}
+                        avatarID={currentUserPersonalDetails?.accountID}
+                        source={currentUserPersonalDetails?.avatar}
+                        fallbackIcon={currentUserPersonalDetails.fallbackIcon}
+                    />
+                    <View style={[styles.flex1, styles.flexShrink1, styles.flexBasis0, styles.justifyContentCenter, styles.gap1]}>
+                        <View style={[styles.flexRow, styles.gap1]}>
+                            <Text
+                                numberOfLines={1}
+                                style={[styles.textBold, styles.textLarge]}
+                            >
+                                {currentUserPersonalDetails?.displayName}
+                            </Text>
+                            {canSwitchAccounts && (
+                                <View style={styles.justifyContentCenter}>
+                                    <Icon
+                                        fill={theme.icon}
+                                        src={Expensicons.CaretUpDown}
+                                        height={variables.iconSizeSmall}
+                                        width={variables.iconSizeSmall}
+                                    />
+                                </View>
+                            )}
+                        </View>
+                        <Text
+                            numberOfLines={1}
+                            style={[styles.colorMuted, styles.fontSizeLabel]}
+                        >
+                            {currentUserPersonalDetails?.login}
+                        </Text>
+                    </View>
+                </View>
+            </PressableWithFeedback>
+            {canSwitchAccounts && (
+                <Popover
+                    isVisible={shouldShowDelegatorMenu}
+                    onClose={() => {
+                        setShouldShowDelegatorMenu(false);
+                        clearDelegatorErrors();
+                    }}
+                    anchorRef={buttonRef}
+                    anchorPosition={styles.accountSwitcherAnchorPosition}
+                >
+                    <View style={styles.pb4}>
+                        <Text style={[styles.createMenuHeaderText, styles.ph5, styles.pb2, styles.pt4]}>{translate('delegate.switchAccount')}</Text>
+                        <MenuItemList
+                            menuItems={menuItems()}
+                            shouldUseSingleExecution
+                        />
+                    </View>
+                </Popover>
+            )}
```

Comment:
The popover content had no safe area (https://github.com/Expensify/App/issues/48292)